Userland/Libraries/LibTLS/Socket.cpp

```diff
@@ -6,35 +6,39 @@
 
 #include <AK/Debug.h>
 #include <LibCore/DateTime.h>
+#include <LibCore/EventLoop.h>
 #include <LibCore/Timer.h>
 #include <LibCrypto/PK/Code/EMSA_PSS.h>
 #include <LibTLS/TLSv12.h>
 
+#define MUST_OR_LOG(x) ({                                                               \
+    auto __temp_result = (x);                                                           \
+    if (__temp_result.is_error()) {                                                     \
+        dbgln("{}:{}: {} -> error: {}", __FILE__, __LINE__, #x, __temp_result.error()); \
+        VERIFY_NOT_REACHED();                                                           \
+    }                                                                                   \
+    __temp_result.release_value();                                                      \
+})
+
```

Comment:
Seems like an interesting addition, does adding it to `AK/Try.h` not work?

Reply:
it does not, and I'm not gonna add it either, sorry.